apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/TabButton/index.tsx

```diff
@@ -1,27 +1,46 @@
-import React, { useCallback } from "react";
-import { useTheme } from "@react-navigation/native";
+import React, { useCallback, useState } from "react";
+import { useTheme, useFocusEffect } from "@react-navigation/native";
 import { View, StyleSheet } from "react-native";
 import { Text } from "@ledgerhq/native-ui";
 import { BorderlessButton } from "react-native-gesture-handler";
 import type { AppProps, MainProps, SearchProps } from "LLM/features/Web3Hub/types";
 import { NavigatorName, ScreenName } from "~/const";
+import deviceStorage from "~/logic/storeWrapper";
+import { Web3HubTabType } from "../../screens/Web3HubTabs/components/TabItem";
 
 type Props = {
-  count: number;
+  count?: number;
+  onclick?: () => void;
```

Comment:
We usually use the camelCase here
```suggestion
  onClick?: () => void;
```